cluster/addons/fluentd-gcp/scaler-deployment.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: kube-system
   labels:
     k8s-app: fluentd-gcp-scaler
-    version: v0.3.0
+    version: v0.2.0
```

Comment:
This just uses older version, how about disabling this addon altogether?

Reply:
Sorry.. changed at the wrong place. Should've changed `--ds-name=...`. Done.